applications/StructuralMechanicsApplication/custom_elements/prestress_membrane_element.cpp

```diff
@@ -868,7 +885,7 @@ void PrestressMembraneElement::CalculateAll(
         CalculateMetricDeformed(PointNumber, DN_De, gab, g1, g2);
         CalculateStrain(StrainVector, gab, mGab0[PointNumber]);
 
-        Vector CartesianStrainVector = prod(Q, StrainVector);
+        Vector CartesianStrainVector = prod(Q, StrainVector); //in refence configuration
```

Comment:
local variables are lower case with underscores, i.e.`cartesian_strain_vector`
See the [Style Guide](https://github.com/KratosMultiphysics/Kratos/wiki/Style-Guide#local-variables)

Reply:
done